### PR TITLE
Add feature

### PR DESCRIPTION
**Pull request by simpaxd**

### CODE REVIEW

File: Print.java
Comment (karlmosen) [inline Print.java:1]: IT would be a good idea to print the line as well.

**Review by karlmosen — CHANGES_REQUESTED** [review]
Print line.

File: Print.java
Comment (simpaxd) [reply Print.java:1]: I agree

Comment (simpaxd) [reply Print.java:1]: Fixed

**Review by karlmosen — APPROVED** [review]
Very nice

**Pull request merged**